lambdas/functions/management/management.go

```diff
@@ -0,0 +1,59 @@
+package management
+
+import (
+	"context"
+	"fmt"
+	"github.com/weka/aws-tf/modules/deploy_weka/lambdas/functions/weka_api"
+
+	"github.com/weka/go-cloud-lib/connectors"
+	"github.com/weka/go-cloud-lib/lib/jrpc"
+	"github.com/weka/go-cloud-lib/lib/weka"
+	"github.com/weka/go-cloud-lib/logging"
+	"github.com/weka/go-cloud-lib/protocol"
+)
+
+func GetWekaJRPC(ctx context.Context, request weka_api.WekaApiRequest) (interface{}, error) {
```

Comment:
This function `CallJRPC`

Reply:
4d60cca8682b53101f94f0e629716b150453599e